packages/genesys-spark-components/src/components/stable/gux-badge/tests/gux-badge.e2e.playwright.ts

```diff
@@ -0,0 +1,37 @@
+import { analyze, setContent, test } from '@test/playwrightTestUtils';
+
+test.describe('gux-badge', () => {
+  [
+    '<gux-badge>Badge</gux-badge>',
+    '<gux-badge accent="info">Badge</gux-badge>',
+    '<gux-badge accent="success">Badge</gux-badge>',
+    '<gux-badge accent="warning">Badge</gux-badge>',
+    '<gux-badge accent="error">Badge</gux-badge>',
+    '<gux-badge accent="inherit">Badge</gux-badge>',
+    '<gux-badge bold>Badge</gux-badge>',
+    '<gux-badge bold accent="info">Badge</gux-badge>',
+    '<gux-badge bold accent="success">Badge</gux-badge>',
+    // '<gux-badge bold accent="warning">Badge</gux-badge>',
+    '<gux-badge bold accent="error">Badge</gux-badge>',
+    '<gux-badge bold accent="inherit">Badge</gux-badge>',
+    '<gux-badge><gux-icon icon-name="subtract" decorative></gux-icon>Badge</gux-badge>',
+    '<gux-badge accent="info"><gux-icon icon-name="subtract" decorative></gux-icon>Badge</gux-badge>',
```

Comment:
Could this be optimised to pass in the props dynamically instead of rendering each individually or would that affect performance ?

Reply:
Ya I have been thinking about optimising this too. I think doing them all as individual tests will end up being too slow but for now I am keeping them the same as the puppeteer tests to make it easy to see we can write equivalent tests. I will add an issue to the epic to look into it.